### PR TITLE
⬆️ Use clippy from rustup

### PR DESCRIPTION
**Pull request by peterfication**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ rust:
   - nightly
 cache: cargo
 before_script:
-  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then (cargo install clippy --force) fi
+  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then (rustup component add clippy-preview) fi
   - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then (cargo install rustfmt-nightly --force) fi
```
[the remark this answers is not in the document]
Comment (peterfication) [reply .travis.yml]: 👍

**Comment by peterfication** [comment]
TODO:
```
$ if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then (cargo fmt -- --write-mode=diff) fi
Unrecognized option: 'write-mode'
The command "if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then (cargo fmt -- --write-mode=diff) fi" exited with 1.
```

**Comment by tomhoule** [comment]
For reference here's what I have in graphql-client:

`  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then (cargo fmt --all -- --check) fi`

**Comment by tomhoule** [comment]
This was done as part of my latest PR, so let's close this one.

**Pull request closed without merging**